include/sound/sof/info.h

```diff
@@ -111,7 +111,7 @@ struct sof_ipc_cc_version {
 
```

Comment:
Changing a packed struct like this requires a better explanation in commit message why this is a safe change to do.
Also the description is a bit too vague. What compiler related issue, and how is a kernel change related to FW ELF generation -- this needs to be explained better.